Sources/LiveViewNative/Stylesheets/ParseableTypes/AnyGesture+ParseableModifierValue.swift

```diff
@@ -0,0 +1,199 @@
+                OnEnded.parser(in: context).map(Self.onEnded)
+                Exclusively.parser(in: context).map(Self.exclusively)
+            }
+        }
+        
+        func apply(to gesture: AnyGesture<Any>) -> AnyGesture<Any> {
+            switch self {
+            case .sequenced(let sequenced):
+                return AnyGesture<Any>(gesture.sequenced(before: sequenced.other).map({ $0 as Any }))
+            case .simultaneously(let simultaneously):
+                return AnyGesture<Any>(gesture.simultaneously(with: simultaneously.other).map({ $0 as Any }))
+            case .onEnded(let onEnded):
+                return AnyGesture<Any>(gesture.onEnded({ value in
+                    Task {
+                        await onEnded.action(value: value)
+                    }
+                }).map({ $0 as Any }))
+            case .exclusively(let exclusively):
+                return AnyGesture<Any>(gesture.exclusively(before: exclusively.other).map({ $0 as Any }))
+            }
+        }
+    }
+    
+    static func baseParser(in context: ParseableModifierContext) -> some Parser<Substring.UTF8View, Self> {
+        OneOf {
+            #if !os(tvOS)
+            Drag.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+            #endif
+            LongPress.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+            #if os(iOS) || os(macOS)
+            Magnify.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+            Rotate.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+            #endif
+            #if !os(tvOS)
+            SpatialTap.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+            #endif
+            Tap.parser(in: context).map({ Self.init($0.value.map({ $0 as Any })) })
+        }
+    }
+    
+    #if !os(tvOS)
+    @ParseableExpression
+    struct Drag {
+        static let name = "DragGesture"
+        let value: DragGesture
+        
+        init(minimumDistance: CGFloat = 10, coordinateSpace: AnyCoordinateSpaceProtocol = .local) {
+            self.value = .init(minimumDistance: minimumDistance, coordinateSpace: coordinateSpace.coordinateSpace)
+        }
+    }
+    #endif
+    
+    @ParseableExpression
+    struct LongPress {
+        static let name = "LongPressGesture"
+        let value: LongPressGesture
+        
+        #if os(tvOS)
+        init(minimumDuration: Double = 0.5) {
+            self.value = .init(minimumDuration: minimumDuration)
+        }
+        #else
+        init(minimumDuration: Double = 0.5, maximumDistance: CGFloat = 10) {
+            self.value = .init(minimumDuration: minimumDuration, maximumDistance: maximumDistance)
+        }
+        #endif
+    }
+    
+    #if os(iOS) || os(macOS) || os(xrOS)
+    @ParseableExpression
+    struct Magnify {
+        static let name = "MagnifyGesture"
+        let value: AnyGesture<Any>
```

Comment:
Would wrapping the entire struct body in the `if #available` conditional allow us to avoid the `AnyGesture` type here by specifying `MagnifyGesture` or `MagnificationGesture` respectively?

Reply:
I don't think it'd be worthwhile, given that the type we're parsing is an erased `AnyGesture` anyways.